build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="Microsoft.DotNet.Cli.Monikers.props" />
```

Comment:
I felt that it was weird too actually, I like using build.proj. Will make the change.